src/homework_8/myArrayList/MyArrayList.java

```diff
@@ -0,0 +1,138 @@
+package homework_8.myArrayList;
+
+import java.util.Arrays;
+
+public class MyArrayList<T> implements List<T> {
+    private static final Object[] EMPTY_ARRAY = {};
+    private static final Object[] FULL_ARRAY_CAPACITY = new Object[10];
+    private T[] array;
+
+    public MyArrayList() {
+        array = (T[]) FULL_ARRAY_CAPACITY;
+    }
+
+    public MyArrayList(int capacity) {
+        if (capacity > 0) {
+            array = (T[]) new Object[capacity];
+        } else if (capacity == 0) {
+            array = (T[]) EMPTY_ARRAY;
+        } else new IllegalArgumentException("Передано невірний аргумент");
+    }
+
+    /**
+     * Окрема реалізація метода add()
+     */
+    @Override
+    public void add(T value) {
```

Comment:
try to simplify this method. If you don't not know how to do it, I  will help you (write me in slack).

better create field:    private int position = 0;   and increment it after adding the elements.